drizzlepac/devutils/comparison_tools/compare_sourcelists.py

```diff
@@ -489,8 +538,10 @@ def getMatchedLists(slNames,imgNames,slLengths):
         log.info("WARNING: Unable to fetch values for xref and yref from fits file headers. Using xref = 0.0 and yref = 0.0.")
         xref=0.0
         yref=0.0
-    log.info("source_list_dict: {}".format(source_list_dict))
-    out_dict = starmatch_hist.run(source_list_dict, xref=xref, yref=yref)
+    log.info("Summary of input catalog lengths")
+    for source_list in source_list_dict.keys():
+        log.info("{}: {}".format(os.path.basename(source_list),source_list_dict[source_list]))
```

Comment:
The log comment indicates that this will be reporting the number of sources (catalog lengths) for all inputs.  However, the actual log message only seems to be reporting the source_list_dict entries.  It is NOT clear that this value is the length, or the actual list of sources (not just the length).  Either additional comments would be helpful or perhaps use of len() on a catalog would be needed to make the code consistent with the log comment.